### ISSUE TITLE
Implement batch_norm

### ISSUE THREAD

**Issue by bbradelTT** [tenstorrent/tt-metal#15372]
pytorch2.0 has a batch_norm. ttnn does not. They have a workaround by breaking down batch_norm into multiple operations. However, it might be good to have a direct ttnn batch_norm call.

Torch documentation:
https://pytorch.org/docs/stable/generated/torch.nn.functional.batch_norm.html

See: https://github.com/tenstorrent/pytorch2.0_ttnn/issues/433 and https://github.com/tenstorrent/pytorch2.0_ttnn/issues/465

**Comment by cmaryanTT** [comment]
@VirdhatchaniKN - can this be closed? Is our Batch norm support reasonably general?

**Comment by VirdhatchaniKN** [comment]
Yes @cmaryanTT 
Our `ttnn.batch_norm` supports the mentioned inputs (as 4D Tensors)

**Comment by VirdhatchaniKN** [comment]
Closing issue as support has been merged to main
